uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/DebuggerTest.java

```diff
@@ -196,12 +195,22 @@ public void testDebugger() {
 		AbstractWait.sleep(TimePeriod.getCustom(5));
 		assertEquals(CHOICE_ID, variables.getValue("Endpoint"));
 
-		// get body of message
-		variables.close();
-		AbstractWait.sleep(TimePeriod.SHORT);
-		variables.open();
-		new DefaultTree().getItems().get(4).getItems().get(0).select();
-		assertTrue(new DefaultStyledText().getText().contains("<name>Erie Zoo</name>"));
+		for (int i = 0; i < 4; i++) {
```

Comment:
Why do you need 4 iterations here?

Reply:
There are five test messages which can be processed in random order and I need to wait for the right message ;-).